src/broker/controller.c

```diff
@@ -64,8 +65,14 @@ static int controller_name_new(ControllerName **namep, Controller *controller, c
 /**
  * controller_name_reset() - XXX
  */
-void controller_name_reset(ControllerName *name) {
-        activation_flush(&name->activation);
```

Comment:
Can you make `activation_flush()` static now? Not used by anyone but activation.c.

Reply:
Done.